lib/hydra/pcdm/models/concerns/object_behavior.rb

```diff
@@ -40,7 +40,7 @@ def pcdm_collection?
     end
 
     def in_objects
-      aggregated_by.select(&:pcdm_object?)
+      ordered_by.select(&:pcdm_object?).to_a
```

Comment:
But don't we want it to be lazy. What If I call in_objects.first, I don't want everything.